apps/web/playwright/fixtures/webhooks.ts

```diff
@@ -12,7 +12,6 @@ export function createWebhookPageFixture(page: Page) {
       await page.waitForURL((u) => u.pathname === "/settings/developer/webhooks/new");
       const url = page.url();
       const teamId = Number(new URL(url).searchParams.get("teamId")) as number;
-      await page.click('[data-testid="new_webhook"]');
```

Comment:
Hmm this is wrong. I don't know how this e2e test passed. `/settings/developer/webhooks/new` is expected to not have this button.